diff -burNp xpad-2.14/xpad.desktop xpad-2.14-new/xpad.desktop
--- xpad-2.14/xpad.desktop	2007-11-07 04:08:52.000000000 +0100
+++ xpad-2.14-new/xpad.desktop	2009-01-22 14:44:14.454542475 +0100
@@ -5,13 +5,15 @@ Name=Xpad
 Name[zh_TW]=Xpad
 GenericName=Sticky Notes
 GenericName[zh_TW]=自粘性便條
+GenericName[pl]=Karteczki
 Comment=Jot down notes for later
-Comment[zh_TW]=大略記下備註以供之後使用
+Comment[zh_TW]=大略記下備
+Comment[pl]=Notatki na później註以供之後使用
 
 TryExec=xpad
 Exec=xpad
 Terminal=false
-Icon=xpad
+Icon=xpad.png
 
 StartupNotify=true
 StartupWMClass=xpad
