diff -urN xpad-2.10/xpad.desktop.in xpad-2.10.new/xpad.desktop.in
--- xpad-2.10/xpad.desktop.in	2004-04-27 00:09:02.000000000 +0000
+++ xpad-2.10.new/xpad.desktop.in	2005-11-01 11:01:38.000000000 +0000
@@ -2,18 +2,21 @@
 Encoding=UTF-8
 Version=0.9.4
 
-_Name=Xpad
+Name=Xpad
 _GenericName=Sticky Notes
+GenericName[pl]=Karteczki
 _Comment=Jot down notes for later
+Comment[pl]=Notatki na później
 
 FilePattern=xpad;
 TryExec=xpad
 Exec=xpad
 Terminal=false
-Icon=xpad
+Icon=xpad.png
 
 StartupNotify=true
 StartupWMClass=xpad
 
 Type=Application
-Categories=Application;Utility;GTK;
+Categories=GTK;Utility;
+# vi: encoding=utf-8
